src/haskell/test/Arion/NixSpec.hs

```diff
@@ -1,6 +1,7 @@
 {-# LANGUAGE OverloadedStrings #-}
 module Arion.NixSpec
   ( spec
+    , contextSpec
```

Comment:
Could you change `spec` to look like the following?

```haskell
spec = describe "evaluateComposition" $ do
  it "matches an example" $ do
    ...
  it "matches a build.context example" $ do
    ...

```

`describe` forms a tree structure, where `it` are the leaves, so we can make use of that here, keep the interface simple.

Reply:
Done in  3588b01